### PR TITLE
Port website to hugo

### PR DESCRIPTION
**Pull request by Steiniche**
We will use this PR to track the status of the current port of our website to hugo:

- [x] Port Head
- [x] Port Header
- [x] Port Index
- [x] Port Footer

Later we would like to also port:
- Meetup Groups
- Speakers
- Sponsors
- Events (should point to the new site at https://community.cncf.io/, so people know where to watch for new events)

The first three should be done with some content / bundles in hugo where a new markdown file produces a new item in the list.
In this way it will be easy to update the site by adding a new markdown file with the information.

### CODE REVIEW

**Comment by Steiniche** [comment]
@Solisol @kaspernissen heads up.

**Comment by kaspernissen** [comment]
This is a great start. 💪

**Comment by Steiniche** [comment]
When doing the review you can view the website locally by running `make run` and visiting `localhost:1313`

File: hugo/layouts/partials/header.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-expand-md navbar-dark fixed-top bg-gradient">
+      <a class="navbar-brand" href="/"><img src="logo.svg" /></a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarCollapse">
```
Comment (kaspernissen) [inline hugo/layouts/partials/header.html:6]: Great with the ability to port the rest of the pages! Can we hide them in the initial release?

File: hugo/layouts/index.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+<section class="bg-purple py-4">
+	<div class="container">
+		<div class="row">
+			<div class="col-sm-12 col-lg-6 align-self-center">
+				<h3>Join our Slack community</h3>
+			</div>
+			<div class="col-sm-12 col-lg-6">
+				<form class="mb-0">
+					<input type="email" class="form-control rounded form-control-lg" placeholder="Enter email..." />
```
Comment (kaspernissen) [inline hugo/layouts/index.html:31]: Did you find a solution to make the request to the slack-invite service? I might have missed it in all the files in this PR and given its 5:30 in the morning 🙈

**Review by kaspernissen — APPROVED** [review]
This is awesome, @Steiniche! Looks exactly like the original. Great job!

File: hugo/layouts/index.html (hunk 2 above)
Comment (Steiniche) [reply hugo/layouts/index.html:31]: No I have not tested this yet, but it should be possible just to put the URL in as a get action and then the slack-invite can use the email in the get params.
So we just need to move the DNS and then we can set this up proper.

File: hugo/layouts/partials/header.html (hunk 1 above)
Comment (Steiniche) [reply hugo/layouts/partials/header.html:6]: Sure we can, will look into it soon.

**Comment by Steiniche** [comment]
I guess we should do some pair programming where we move the DNS and everything and then put the new site into production.
Lets discuss on slack when people have time.

**Comment by Steiniche** [comment]
@kaspernissen would it make sense to merge this now as it will land in /hugo and then create another PR for the finale change where we remove the "old" website and put the "new" website online?

**Comment by kaspernissen** [comment]
Yes, I think that would make sense! 

Let's schedule a hack night again, maybe next week? 😄

**Comment by Steiniche** [comment]
@kaspernissen cool then this can be merged.

Yeah lets continue the talk on slack :+1:

**Pull request merged**